src/lib/checksum.lua

```diff
@@ -10,6 +10,7 @@ require("lib.checksum_h")
 local lib = require("core.lib")
 local ffi = require("ffi")
 local C = ffi.C
+local band = bit.band
 
 -- Select ipsum(pointer, len, initial) function based on hardware
```

Comment:
Comment here seems like the right place to say that we expect network byte order?